CHANGELOG.md

```diff
@@ -6,23 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Features
+## [v2.0.0-beta1](https://github.com/regen-network/regen-ledger/releases/tag/v2.0.0-beta1) - 2021-08-18
 
-* Add support for on-chain creation of Permanent Locked Accounts (#482, [regen-network/cosmos-sdk#42](http://github.com/regen-network/cosmos-sdk/pull/42)
+### General
+
+#### Added
+
+* [#388](https://github.com/regen-network/regen-ledger/pull/388) Add support for rosetta
+* ([#482](https://github.com/regen-network/regen-ledger/pull/482),
+    [regen-network/cosmos-sdk#42](http://github.com/regen-network/cosmos-sdk/pull/42))
+    Add support for on-chain creation of Permanent Locked Accounts
```

Comment:
This should be one line.
```suggestion
* ([#482](https://github.com/regen-network/regen-ledger/pull/482), [regen-network/cosmos-sdk#42](http://github.com/regen-network/cosmos-sdk/pull/42)) Add support for on-chain creation of Permanent Locked Accounts
```

Reply:
Another option: might be nice to use one line and put the fork link at the end.
```suggestion
* [#482](https://github.com/regen-network/regen-ledger/pull/482) Add support for on-chain creation of Permanent Locked Accounts [regen-network/cosmos-sdk#42](http://github.com/regen-network/cosmos-sdk/pull/42)
```

Reply:
These still render as a single line in the Markdown view. The multiline is only so that we keep line-width from being too long when viewing in an editor (so to make writing/editing of the changelog easier). I think I'd like to keep it as multiline, but like your suggestion of putting the fork PR at the end.